Content.Shared/VendingMachines/SharedVendingMachineSystem.cs

```diff
@@ -1,10 +1,12 @@
+using Content.Shared.Emag.Systems;
 using Robust.Shared.Prototypes;
 using System.Linq;
 
 namespace Content.Shared.VendingMachines;
 
 public abstract class SharedVendingMachineSystem : EntitySystem
 {
+    [Dependency] protected readonly EmagSystem _emagSystem = default!;
```

Comment:
is making it protected ok or should this be private + have another in server VendingMachineSystem